09_dt_rtc/log.txt

```diff
@@ -0,0 +1,99 @@
+     0  1  2  3  4  5  6  7  8  9  a  b  c  d  e  f                             
+00:          -- -- -- -- -- -- -- -- -- -- -- -- --                             
+10: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --                             
+20: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --                             
+30: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --                             
+40: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --                             
+50: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --                             
+60: -- -- -- -- -- -- -- -- UU -- -- -- -- -- -- --                             
+70: -- -- -- -- -- -- -- --                                                     
+/ # date                                                                        
+Sat Jan  1 00:18:43 UTC 2000                                                    
+/ # date -s "2021-03-25 1:16"                                                   
+Thu Mar 25 01:16:00 UTC 2021                                                    
+/ # date                                                                        
+Thu Mar 25 01:16:33 UTC 2021                                                    
+/ # hwclock                                                                     
+Sat Jan  1 00:21:34 2000  0.000000 seconds                                      
+/ # hwclock -w                                                                  
+/ # hwclock -w                                                                  
+/ # hwclock                                                                     
+Thu Mar 25 01:18:09 2021  0.000000 seconds                                      
+/ # i2cdetect -r -y 0                                                           
+     0  1  2  3  4  5  6  7  8  9  a  b  c  d  e  f                             
+00:          -- -- -- -- -- -- -- -- -- -- -- -- --                             
+10: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --                             
+20: -- -- -- -- UU -- -- -- -- -- -- -- -- -- -- --                             
+30: -- -- -- -- 34 -- -- -- -- -- -- -- -- -- -- --                             
+40: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --                             
+50: UU -- -- -- -- -- -- -- -- -- -- -- -- -- -- --                             
+60: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --                             
+70: 70 -- -- -- -- -- -- --                                                     
+/ # cat sys/bus/i2c/devices/0-0050/eeprom | hexdump -C                          
+00000000  aa 55 33 ee 41 33 33 35  42 4e 4c 54 30 30 43 30  |.U3.A335BNLT00C0|  
+00000010  32 34 31 39 42 42 42 4b  30 31 42 42 ff ff ff ff  |2419BBBK01BB....|  
+00000020  ff ff ff ff ff ff ff ff  ff ff ff ff ff ff ff ff  |................|  
+*                                                                               
```

Comment:
According to task description:
1. you should use EEPROM on additional board - i2c2 bus 
2. you should make change to devicetree to override EEPROM model to the one that you use

Reply:
I don't have the same RTC as in the task that is why I have task with BeagleBone EEPROM